spec/spec_helper_acceptance.old

```diff
@@ -0,0 +1,188 @@
+    # Omitting datadir and data_hash to use defaults.
+
+  - name: "Other YAML hierarchy levels"
+    paths: # Can specify an array of paths instead of one.
+      - "location/%{facts.whereami}/%{facts.group}.yaml"
+      - "groups/%{facts.group}.yaml"
+      - "os/%{facts.os.family}.yaml"
+      - "%{facts.os.family}.yaml"
+      - "#{vmhostname}.yaml"
+      - "Redhat.yaml"
+      - "common.yaml"
+EOS
+
+
+        if fact('osfamily') == 'Debian'
+          runtime = 'cri_containerd'
+          cni = 'weave'
+          #Installing rubydev environment
+          on(host, "apt install ruby-bundler --yes", acceptable_exit_codes: [0]).stdout
+          on(host, "apt-get install ruby-dev --yes", acceptable_exit_codes: [0]).stdout
+          on(host, "apt-get install build-essential curl git m4 python-setuptools ruby texinfo libbz2-dev libcurl4-openssl-dev libexpat-dev libncurses-dev zlib1g-dev --yes", acceptable_exit_codes: [0]).stdout
+          on(host, "gem install bundler", acceptable_exit_codes: [0]).stdout
+        end
+        if fact('osfamily') == 'RedHat'
+          runtime = 'docker'
+          cni = 'flannel'
+          #Installing rubydev environment
+          on(host, "yum install -y ruby-devel git zlib-devel gcc-c++ lib yaml-devel libffi-devel make bzip2 libtool curl openssl-devel readline-devel", acceptable_exit_codes: [0]).stdout
+          on(host, "gem install bundler", acceptable_exit_codes: [0]).stdout
+          on(host, "setenforce 0 || true", acceptable_exit_codes: [0]).stdout
+          on(host, "swapoff -a", acceptable_exit_codes: [0]).stdout
+          on(host, "systemctl stop firewalld && systemctl disable firewalld", acceptable_exit_codes: [0]).stdout
+          on(host, "yum install -y yum-utils device-mapper-persistent-data lvm2", acceptable_exit_codes: [0]).stdout 
+          on(host, "yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo", acceptable_exit_codes: [0]).stdout 
+          on(host, "yum install -y docker-ce-18.06.3.ce-3.el7", acceptable_exit_codes: [0]).stdout 
+          on(host, "usermod -aG docker $(whoami)", acceptable_exit_codes: [0]).stdout 
+          on(host, "systemctl enable docker.service", acceptable_exit_codes: [0]).stdout 
+          on(host, "sudo systemctl start docker.service", acceptable_exit_codes: [0]).stdout
+          if fact('operatingsystem') != 'RedHat'
+            on(host, "yum install -y epel-release", acceptable_exit_codes: [0]).stdout
+          end
+          if fact_on(host, 'operatingsystem') == 'RedHat'
+            on(host, 'mv /etc/yum.repos.d/redhat.repo /etc/yum.repos.d/internal-mirror.repo')
+            on(host, 'rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm')
+          end
+          on(host, 'yum update -y -q') if fact_on(host, 'osfamily') == 'RedHat'
+          on(host, "yum install -y python-pip", acceptable_exit_codes: [0]).stdout
```

Comment:
@carabasdaniel too good. Minor comment. We can remove the line number 149,150 and 151, since it's not needed anymore. This change is added to the helm and rook module.

Reply:
@sheenaajay Sorry about leaving that in, just removed the spec_helper_acceptance.old file, that was a backup of the original acceptance helper